proof/refine/ARM/IpcCancel_R.thy

```diff
@@ -112,6 +112,7 @@ lemma cancelSignal_st_tcb_at':
                     hoare_drop_imp[where R="\<lambda>rv _. \<exists>a b. delete t (f rv) = a # b" for f])
   done
 
+(* Not used?
```

Comment:
@michaelmcinerney did you still need this?

Reply:
Yes this is needed for `suspend_invs'`. I'll update this proof to use this soon

Reply:
Ok, so, it would be ok to leave these for you to fix up with your `suspend_invs’` proof, right? @michaelmcinerney ?

Reply:
Sure. I can make a PR just for this, but I'd prefer to just include it next time I make a PR for something else

Reply:
Thanks, I’ll add some comments to note that. I think it should be ok if you include these with your `suspend_ivns’` proof.

Reply:
If you're sure @michaelmcinerney is going to uncomment it again in the next PR I'm fine without a `FIXME RT`, but if there's a chance we might forget, I'd still tag it.